tests/ara/test_job_client.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from citrine.resources.ara_job import TaskNode, JobStatusResponse
+
+
+def task_node_1() -> dict:
+    tn1 = {'id': 'dave_id1', 'task_type': 'dave_type', 'status': 'dave_status',
+                   'dependencies': {'dep1', 'dep2'}}
```

Comment:
I see at least one test failure is a result of you omitting a `failure_reason` of `None` from the dump

Reply:
See you fixed that. Looks like the remaining issues are all test coverage / flake8 appeasement